src/components/AccountCircleButton.tsx

```diff
@@ -20,26 +19,5 @@ export default function AccountCircleButton({ style, size, testID }: Props) {
     navigate(Screens.ProfileMenu)
   }
 
-  return (
-    <View style={styles.container}>
-      <Touchable
-        testID={testID}
-        onPress={onPress}
-        style={[style, styles.button]}
-        borderRadius={Spacing.Thick24}
-      >
-        <AccountCircle size={size} />
-      </Touchable>
-    </View>
-  )
+  return <TopBarIconButtonV2 icon={<AccountCircle />} testID={testID} onPress={onPress} />
```

Comment:
Should style be passed as a prop to `TopBarIconButtonV2`? It look like it takes it as a prop. Also should `<AccountCircle size={size} />` be used for the icon (including size like it was before)?

Reply:
Thanks, added